release_notes/ocp-4-5-release-notes.adoc

```diff
@@ -2672,3 +2672,19 @@ link:https://access.redhat.com/solutions/6075411[{product-title} 4.5.40 containe
 ==== Upgrading
 
 To upgrade an existing {product-title} 4.5 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#update-service-overview_updating-cluster-cli[Updating a cluster by using the CLI for instructions.]
+
+[id="ocp-4-5-41"]
+=== RHBA-2021:2431 - {product-title} 4.5.41 bug fix and security update
```

Comment:
Usually use the images ID for this

Reply:
Ahh yeah. I always do, but somehow this one was deleted when I swapped off the branch. I guess I put 2431 on accident when I re-did it.